_summaries/mescheder2018occupancy_1.md

```diff
@@ -1,19 +1,66 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: mescheder2018occupancy
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: ishanashah
+score: 10
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+Problem: Need a good representation of 3D data for deep learning.
+
+## Related Work
+* Voxels: 3D pixels, but memory requirements grow cubically with size.
+* Point Clouds: Don't have connectivity information between points, so difficult to extract surfaces.
+* Meshes: Usually based on a template mesh, because it is very difficult to handle meshes of very different objects.
+
+![FIGURE_1](mescheder2018occupancy_1a.PNG)
+
+## Solution
+Reason about occupancy not only at fixed discrete 3D locations, but at any possible 3D point.
+
+Learn a nonlinear binary classifier that can determine whether any given 3D point is inside or outside the object.
+
+The function is called an occupancy network, because it predicts the probability that a point is occupied.
+
+The surface of points where the probability of occupancy is 0.5 is the surface of the object.
+
+For 3D reconstruction, the occupancy network takes a pair of the input image and 3D location as inputs.
+
+## Training and Inference
+
+Occupancy network is trained on a random sample of points with known labels.
```

Comment:
Could add some details on the occupancy network implementation details: (fully connected network with 5 ResNet blocks conditioned on the input using conditional batch normalization).